Kudu.Contracts/Functions/IKeyJsonOps.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public interface IKeyJsonOps<T>
+    {
+        int GetKeyNumbers();
```

Comment:
This would probably be more appropriately named as something like `RequiredKeyCount` and as a property, instead of a method.